staging/publishing/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- sttts
+- nikhita
+- dims
+reviewers:
+- sttts
+- dims
+- nikhita
```

Comment:
@sttts can you also add the labels config here so that it automatically gets added?

```
labels:
- sig/release
- area/release-infra
```

Reply:
done